wiki/People/Featured_Artists/de.md

```diff
@@ -0,0 +1,130 @@
+---
+tags:
+  - FA
+---
+
+# Gesponserte Künstler
+
+**Gesponserte Künstler** (auch *Featured Artists* oder kurz ***FA*** genannt) sind Musikschaffende, die mit osu! zusammenarbeiten, um einige ihrer Songs für die Verwendung in [Beatmaps](/wiki/Beatmap) und als Hintergrundmusik in den Livestreams des [osu! World Cup](/wiki/Tournaments#osu!-world-cup) zur Verfügung zu stellen.
+
+Die [Auflistung der Featured Artists](https://osu.ppy.sh/beatmaps/artists) auf der Webseite präsentiert jeden einzelnen Künstler, der an dem Programm teilnimmt. Jede Künstlerseite enthält allgemeine Informationen über die Musik und eine Liste der Songs, die für die Verwendung in osu! lizenziert sind, zusammen mit vorkonfigurierten `.osz`-Dateien zur Erstellung von Beatmaps.
+
+Das [osu!-Team](/wiki/People/osu!_team) arbeitet manchmal mit Künstlern zusammen, um neue Exklusivtitel für osu! zu produzieren, genannt [osu! originals](/wiki/osu!_originals). Diese werden als solche im FA-Katalog gekennzeichnet.
+
+## Auswahl
+
+Neuzugänge zum Featured-Artist-Programm werden indirekt von der osu!-Community bestimmt. Wenn Mapper Beatmaps für einen Künstler entwerfen und ranken, spricht das osu!-Team den Künstler möglicherweise an, um auf das Featured-Artist-Programm aufmerksam zu machen. Gelegentlich werden Künstler jedoch basierend auf anderen Faktoren zum Featured-Artist-Programm eingeladen, wie beispielsweise ihre Präsenz in der Mapping-Community von osu!.
+
+Alle Songs im Featured-Artist-Programm werden nicht-exklusiv für die Verwendung in osu! lizenziert. Weitere Details für Musikschaffende, die dem Künstlerprogramm beitreten möchten, können im Artikel [Musiklizenzierung](/wiki/Legal/Music_licensing) nachgelesen werden.
+
+## Bekanntgabe neuer gesponserter Künstler
+
+Einbindungen neuer gesponserter Künstler pro Jahr:
+
+- **2016:** 8
+- **2017:** 11
+- **2018:** 12
+- **2019:** 31
+- **2020:** 65
+- **2021:** 101
+- **2022:** 104
+- **2023:** 64
+- **2024:** *noch ausstehend*
+
+Neue gesponserte Künstler werden wöchentlich im [Newsfeed](https://osu.ppy.sh/home/news) von osu! bekanntgegeben. Die meisten Ankündigungen enthalten eine Liste der lizenzierten Songs des Künstlers und einen Link zur Künstlerseite. Einige Ankündigungen sind auch mit Mapping-Wettbewerben oder von der [Mappers' Guild](/wiki/Community/Mappers_Guild) erstellten Beatmaps gepaart.
```

Comment:
We dont want to translate `Mapper's Guild`, do we?

Mappergilde/Mappinggilde 🤔